search_agg_terms.go

```diff
@@ -27,11 +47,12 @@ type TermsAggregation struct {
 }
 
 // NewTermsAggregation instantiates a TermsAggregation targeting the provided field
-// and sets the Size to -1 to be omitted for the default value.
+// Sets Size and MinDocCount to -1 to be omitted for the default value.
```

Comment:
Add a period after "field" in the line above, since godoc will strip the line break and the docs will read as "...field Sets Size ...".